bcda/web/router_test.go

```diff
@@ -48,18 +48,18 @@ func (s *RouterTestSuite) TestDefaultRoute() {
 }
 
 func (s *RouterTestSuite) TestDefaultProdRoute() {
-	err := os.Setenv("DEPLOYMENT_TARGET","prod")
+	err := os.Setenv("DEPLOYMENT_TARGET", "prod")
 	if err != nil {
-		s.FailNow("err in setting env var",err)
+		s.FailNow("err in setting env var", err)
 	}
-
+	// Need a new router because the one in the test setup is not right.
```

Comment:
Confused by this comment. Can we fix what's not right about it? If not, can we clarify the issue here?

Reply:
When the test is set up the variable is not set correctly because it gets set in the body of this test.  Therefore, in the running of this test after the variable is set it needs to create a new router.  This comment isn't strictly necessary, but github was not letting me resolve a conflict without making a change to the file so I elected to make this comment because it stumped Terrance and I why the test was failing and I thought it might be instructive to future devs.